frontends/election-manager/src/app.test.tsx

```diff
@@ -547,7 +548,22 @@ test('tabulating CVRs', async () => {
 
   fireEvent.click(getByText('Close'));
 
+  // Check that a zero report can be generated on the L&A tab even after CVRs have been tabulated
+  fireEvent.click(getByText('L&A'));
+  fireEvent.click(
+    getByText(
+      'Print Pre-Election Unofficial Full Election Tally Report (Zero Report)'
+    )
```

Comment:
While I was just copying the style of surrounding tests, I didn't know about the query priority you linked to! Thanks for sharing that

Reply:
Ooh, so an interesting finding here. `getByRole('button', { name: 'Print...' })` doesn't work because the `LinkButton` component sets the `role` to `option`.

https://github.com/votingworks/vxsuite/blob/a354ea74fb27937b33897a1f7fcf775fdb5fb5fb/frontends/election-manager/src/components/link_button.tsx#L38

https://github.com/votingworks/vxsuite/blob/a354ea74fb27937b33897a1f7fcf775fdb5fb5fb/libs/ui/src/link_button.tsx#L41

Interestingly, `getByRole('option', { name: 'Print...' })` doesn't work either.

There does seem to be some intention behind setting the `role` to `option`:

<img width="950" alt="history" src="https://user-images.githubusercontent.com/12616928/164090797-e5da1fc9-9692-41b3-8b54-618c3ab49949.png">

Reply:
It does seem like there may be an accessibility issue here if `getByRole('button', { name: 'Print...' })` isn't working, but fixing it also seems to be out of scope for this PR. Let me open a GitHub issue to track

Reply:
Oh yeah I ran into this issue recently as well. It probably requires a deeper dive, but for now I just used getByText in these cases. Not ideal but not worth blocking on

Reply:
Opened an issue to track: https://github.com/votingworks/vxsuite/issues/1713